action.yaml

```diff
@@ -229,8 +229,9 @@ runs:
         else
             echo "| Sonar Scan | :${{ steps.all_icons.outputs.sonar_scan }}: |" >> $GITHUB_STEP_SUMMARY
         fi
-
-        if [ "${{ github.event_name }}" == "push" ]; then
+             
+        echo "Blackduck icon '${{ steps.all_icons.outputs.blackduck_scan }}'"
+        if [ "${{ github.event_name }}" == "push" ] || [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
```

Comment:
Same question has similar previous comment.

Reply:
In this case, if we use the condition
` [ "${{ github.event_name }}" == "push" ] || [ "${{ github.event_name }}" != "pull_request" ], `the Slack notification and summary will only work for the "push" and "pull_request" events. Also Slack notifications and GH summary for "pull_request" events should only include "build," "unit_test," and "Sonar scan" steps. Therefore, there is no need to add the "pull_request" condition to this specific configuration.

Reply:
🤖